malio-processor/src/main/java/com/github/nalukit/malio/processor/scanner/AbstractScanner.java

```diff
@@ -0,0 +1,14 @@
+package com.github.nalukit.malio.processor.scanner;
+
+import com.github.nalukit.malio.processor.util.ProcessorUtils;
+
+import javax.lang.model.util.Elements;
+import javax.lang.model.util.Types;
+
+public class AbstractScanner {
```

Comment:
AbstractScanner ist nicht abstract. Gewollt?

Reply:
Oh yes, you are right ... :-)